spec/lib/pre_assembly/bundle_spec.rb

```diff
@@ -77,94 +73,29 @@
     end
   end
 
-  describe 'object discovery: #discover_objects' do
-    let(:tests) do
-      [
-        [:proj_revs,   3, 1, 1],
-        [:proj_rumsey, 3, 2, 2],
-        [:folder_manifest, 3, 2, 2],
-        [:sohp_files_only, 2, 9, 9],
-        [:sohp_files_and_folders, 2, 25, 40]
-      ]
-    end
-
-    it "finds the correct N objects, stageables, and files" do
-      allow_any_instance_of(BundleContextTemporary).to receive(:validate_usage) # req'd for :sohp_files_and_folders
-      tests.each do |proj, n_dobj, n_stag, n_file|
-        b = described_class.new(context_from_proj(proj))
-        b.discover_objects
-        dobjs = b.digital_objects
-        expect(dobjs.size).to eq(n_dobj)
-        dobjs.each do |dobj|
-          expect(dobj.stageable_items.size).to eq(n_stag)
-          expect(dobj.object_files.size).to eq(n_file)
-        end
-      end
+  describe '#digital_objects' do
+    it "finds the correct number of objects" do
+      b = bundle_setup(:folder_manifest)
+      expect(b.digital_objects.size).to eq(3)
```

Comment:
was this because it read nicer, because tests broke with the changing project config requirements against the old test fixtures, or something else? on the one hand, i'm not wild about the cleverness of the old version, since it takes a moment to figure out, and obvious tests are nice. but what about testing all the same fields, for a couple of the original projects, in the more direct style of the new test?  unless it's because the old test fixtures are bogus, in which case, whatever (though i guess i'd still ask if it's worth testing `stageable_items.size` and `object_files.size` on the digital objects for the one project you have in the test).

Reply:
Yeah, only the fixture with a manifest is testable in this way.  Also, the interior of the loop was not actually testing *this* class, but `DigitalObject`.  They weren't really germane to this spec.